Makefile

```diff
@@ -27,6 +27,7 @@ PROJECTS := banyand
 clean: TARGET=clean
 clean: PROJECTS:=$(PROJECTS) pkg
 clean: default  ## Clean artifacts in all projects
+	find . -type s -name 'localhost:*' -delete
```

Comment:
I recently found that `Ginkgo` provides a random port API. I will polish this behavior soon.

Reply:
I'm suffering a massive volume of localhost socket files.